### ISSUE TITLE
1.0.6 shouldn't be marked 'latest stable'

### ISSUE THREAD

**Issue by atlasQED** [aspnet/Microsoft.AspNet.TelemetryCorrelation#66]
1.0.6 is marked as 'latest stable', but depends on a preview version of System.Diagnostics.DiagnosticSource (>= 4.6.0-preview8.19405.3)

**Comment by lmolkova** [comment]
you are right, this is a bug and we'll be fixing it as soon as DiagnosticSource becomes stable
